addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -13,6 +13,70 @@ import { pivotTimeAdapter } from "./pivot_time_adapters";
 const { toString, toNumber, toBoolean } = helpers;
 const { DEFAULT_LOCALE } = constants;
 
+class PivotMeasure {
+    constructor(metaData, fieldName) {
```

Comment:
some jsDoc types ? :roll_eyes: 
at least for the "easy" ones, maybe not a full and complete type for `metaData`